dotnet/Services/ProductReviewService.cs

```diff
@@ -329,15 +331,15 @@ public async Task<ReviewsResponseWrapper> GetReviewsByProductId(string productId
             }
 
             AppSettings settings = await GetAppSettings();
-            if (settings.RequireApproval)
-            {
-                searchQuery = $"{searchQuery} AND approved=true";
-            }
 
-            if (pastReviews && !string.IsNullOrEmpty(locale))
+            if (pastRevNLocale)
             {
+                if (settings.RequireApproval)
+                {
+                    searchQuery = $"{searchQuery} AND approved=true";
```

Comment:
As we discussed for the previous PR, the storefront doesn't have the search feature for now, so it would be unnecessary to add the _searchQuery_ here.